src/box/txn_limbo.c

```diff
@@ -444,14 +460,30 @@ txn_limbo_write_confirm(struct txn_limbo *limbo, int64_t lsn)
 	txn_limbo_write_synchro(limbo, IPROTO_RAFT_CONFIRM, lsn, 0, NULL);
 }
 
+/**
+ * Updates confirmed_entry in case confirmation has occurred.
+ * Must be called strictly after the confirm has actually occurred,
+ * (after the entries have been removed from the limbo).
+ */
+void
+txn_limbo_update_confirmed_entry_after_confirm(struct txn_limbo *limbo,
+					       int64_t lsn)
+{
+	if (lsn >= limbo->confirmed_entry->lsn) {
+		limbo->confirmed_entry = &limbo->head_entry;
+		limbo->confirmed_entry_is_valid = false;
+	}
+}
+
```

Comment:
Given this function is called in a single place of a single cpp file and is just 3 lines long, I suggest to simply inline it there.

Reply:
Fixed.